bindings/nodejs/lib/wallet/wallet.ts

```diff
@@ -145,6 +148,17 @@ export class Wallet {
         return this.methodHandler.getClient();
     }
 
+    /**
+     * Get chrysalis data.
+     */
+    async getChrysalisData(): Promise<Record<string, string>> {
```

Comment:
Delete

Reply:
Remove locally, no need to further comment about it